<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Oscilloscipe D1052E v0.02</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,20">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="NON_LBL_TITLE1">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(0, 0, 0); color:rgb(245, 141, 27);</string>
       </property>
       <property name="text">
        <string>RIGOL</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="NON_LBL_TITLE2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(245, 141, 27);</string>
       </property>
       <property name="text">
        <string> MSO7054</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="20,1">
     <item>
      <widget class="QGroupBox" name="GBOX_waveform">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 255, 255)</string>
       </property>
       <property name="title">
        <string>Waveform</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="10,1">
        <item>
         <widget class="QLabel" name="LBL_PLOT">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>350</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="5">
           <widget class="QLabel" name="LBL_CH_FREQ">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(0, 0, 0)</string>
            </property>
            <property name="text">
             <string>CH Freq</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="NON_LBL_FREQ">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>CH Freq:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="LBL_CH_VAVG">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(0, 0, 0)</string>
            </property>
            <property name="text">
             <string>CH Vavg</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="NON_LBL_VPEAK">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>CH Vpp:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="NON_LBL_VAVG">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>CH Vavg:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="LBL_CH_VPP">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(0, 0, 0)</string>
            </property>
            <property name="text">
             <string>CH Vpp</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QGroupBox" name="GBOX_channel">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 255, 255)</string>
           </property>
           <property name="title">
            <string>Channel</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QComboBox" name="CBOX_channel">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <item>
               <property name="text">
                <string>CH1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CH2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CH3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CH4</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BTN_DB">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Franklin Gothic Demi Cond</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>gui_save.ico</normaloff>gui_save.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>55</height>
          </size>
         </property>
         <property name="title">
          <string>Connection</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>191</width>
            <height>32</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,4">
           <item>
            <widget class="QLabel" name="STAT_CON">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 0, 0);
color: red;
</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="text">
              <string>Discon</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BTN_CON">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="GVOX_plot_config">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>90</height>
          </size>
         </property>
         <property name="title">
          <string>Plot Config</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>191</width>
            <height>61</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout" columnstretch="2,4">
           <item row="0" column="0">
            <widget class="QLabel" name="LBL_VMAX">
             <property name="text">
              <string>Vmax</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="VAL_VMAX">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="maximum">
              <double>10.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="LBL_VMIN">
             <property name="text">
              <string>Vmin</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="VAL_VMIN">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-10.000000000000000</double>
             </property>
             <property name="maximum">
              <double>0.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>-1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QGroupBox" name="GBOX_Operation">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>66</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>66</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 255, 255)</string>
           </property>
           <property name="title">
            <string>Operation</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
            <item>
             <widget class="QPushButton" name="BTN_run">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>RUN</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BTN_auto">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>gui_auto.ico</normaloff>gui_auto.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BTN_DB</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>pressedSaveButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>772</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_run</sender>
   <signal>clicked(bool)</signal>
   <receiver>Form</receiver>
   <slot>pressedRunButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>663</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>936</x>
     <y>674</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_auto</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>pressedAutoSet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>782</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>1183</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CBOX_channel</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>changedChannel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>702</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>1027</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VAL_VMAX</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>changedVplotMinMax()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>780</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>795</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VAL_VMIN</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>changedVplotMinMax()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>780</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>796</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_CON</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>pressedConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>802</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pressedSaveButton()</slot>
  <slot>changedSliderBar()</slot>
  <slot>pressedRunButton()</slot>
  <slot>pressedAutoSet()</slot>
  <slot>changedChannel()</slot>
  <slot>changedVplotMinMax()</slot>
  <slot>pressedConnect()</slot>
 </slots>
</ui>
